src/app.ts

```diff
@@ -40,6 +42,8 @@ app.use(function (req, res, next) {
   res.locals.asset_path = '/public/'
   res.locals.serviceName = 'Example service' // TODO replace with service name
   res.locals.releaseVersion = 'v0.1' // TODO replace with package.json version
+  res.locals.csrfToken = req.csrfToken()
```

Comment:
is this actually validating the token used in posts? You can test by using chrome dev tools to either clear the CSRF field or copy and edit the form post request as curl and repeat (simulating a repeated request).  See [here](https://github.com/ministryofjustice/apvs-external-web/blob/develop/app/app.js#L155) for how this was added to APVS.

Reply:
yep, included details in readme if you want to test this out yourself locally